packages/zowe-explorer/src/trees/job/JobFSProvider.ts

```diff
@@ -225,18 +227,23 @@ export class JobFSProvider extends BaseProvider implements vscode.FileSystemProv
         await AuthHandler.waitForUnlock(spoolEntry.metadata.profile);
         const query = new URLSearchParams(uri.query);
         let recordRange = "";
+        const recordsToFetch = SettingsConfig.getDirectValue<number>("zowe.jobs.paginate.recordsToFetch") ?? 0;
+
         if (query.has("startLine")) {
             const startLine = parseInt(query.get("startLine")!);
-            const endLine = query.has("endLine")
-                ? (parseInt(query.get("endLine")!))
-                : (startLine + (SettingsConfig.getDirectValue<number>("zowe.jobs.paginate.recordsToFetch") ?? 0)) - 1;
+            const endLine = query.has("endLine") ? parseInt(query.get("endLine")!) : startLine + (recordsToFetch - 1);
+            console.log(endLine);
```

Comment:
Looks like there are `console.log` statements on line 235, 237 and 262 from testing